packages/neutron-subdao-timelock-single/src/types.rs

```diff
@@ -53,3 +53,18 @@ impl std::fmt::Display for ProposalStatus {
 pub struct ProposalListResponse {
     pub proposals: Vec<SingleChoiceProposal>,
 }
+
+/// A list of proposals returned by `ListProposals`.
+#[derive(Serialize, Deserialize, Clone, JsonSchema, Debug)]
+pub struct FailedProposalErrors {
+    pub errors: Option<Vec<FailedExecutionError>>,
```

Comment:
I suppose `errors` will be null in json response, if there are no errors found for a proposal?
I think an empty array is more convenient response than a possible `null` value.